mycroft/skills/phillips_hue/test/intent/sample7.intent.json

```diff
@@ -0,0 +1,7 @@
+{
+  "utterance": "activate scene relax",
+  "intent_type": "ActivateSceneIntent",
+  "intent": {
+    "ActivateSceneKeyword": "activate scene"
+  }
+}
```

Comment:
Please, add two intent tests for each intent to provide more coverage for your vocab. Also, for this test, you'll need to add a new `Scene` category in the `intent` section once you make the regex changes.